tox.ini

```
@@ -0,0 +1,29 @@
+[tox]
+requires =
+    tox>=4
+env_list = unit, integration
+
+[testenv]
+skip_install = True
+description = base environment for testing
+deps =
+    pytest>=8
+    numpy>=1.13.3,<2
+    scipy<1.15.0
+    mpi4py<4
+commands =
+    pipinstall --no-deps
+    pytest {posargs:tests}
```

```suggestion
commands =
    pipinstall --no-deps
    pytest {posargs:tests}
```
Are these lines actually needed?

---

Fixed in 6628bb81b05a107c68b2e9c0b5893fefba81bc9a